appveyor.yml

```diff
@@ -4,6 +4,7 @@
   branches:
     only:
       - /v\d\.\d\.\d/
+    except:
```

Comment:
Does it mean it won’t run on master after the merge happens ?

Reply:
It will run on PR's and tags but if I understand correctly not run on master once someone hits the merge button.  Seems to just be an optimization to prevent unnecessary runs in appveyor.

Reply:
I want it to run on master.

Reply:
Made the adjustment, it will now run on master